gusb/gusb-device.c

```diff
@@ -327,6 +329,10 @@ _g_usb_device_save(GUsbDevice *self, JsonBuilder *json_builder, GError **error)
 		json_builder_set_member_name(json_builder, "PlatformId");
 		json_builder_add_string_value(json_builder, priv->platform_id);
 	}
+	if (priv->created != 0) {
+		json_builder_set_member_name(json_builder, "Created");
+		json_builder_add_int_value(json_builder, priv->created);
```

Comment:
what about instead to use g_date_time_format_iso8601 ? It is more readable and you get better information to parse it

Reply:
Hmm, good idea.

Reply:
Something like this^^